docs/TROUBLESHOOTING.md

```diff
@@ -1,80 +1,76 @@
-<h1 align="center">Troubleshooting<a id="troubleshooting"></a></h1>
+# Troubleshooting 
 
 If you need help, first search if there is [already an issue filed](https://issues.redhat.com/issues/?jql=project%20%3D%20OADP) 
   or please log into jira and create a new issue in the `OADP` project.
 
 1. [OADP Cheat Sheet](oadp_cheat_sheet.md)
-2. [Debugging Failed Backups](#backup)
-3. [Debugging Failed Restores](#restore)
-4. [Debugging Data Mover (OADP 1.2 or below)](https://github.com/migtools/volume-snapshot-mover/blob/master/docs/troubleshooting.md)
-5. [Common Issues and Misconfigurations](#misconfig)
-    1. [Credentials Not Properly Formatted](#creds)
-    2. [Errors in the Velero Pod](#velpod)
-    3. [Errors in Backup Logs](#backuplogs)
-    4. [Backup/Restore is Stuck In Progress](#stuck)
-    5. [Restic - NFS Volumes and rootSquash](#rootsquash)
-    6. [Issue with Backup/Restore of DeploymentConfig using Restic](#deployconfig)
-    7. [New Restic Backup Partially Failing After Clearing Bucket](#resbackup)
-    8. [Restic Restore Partially Failing on OCP 4.14 Due to Changed PSA Policy](#psapolicy)
-6. [OADP FAQ](#faq)
-7. [OpenShift ROSA STS and OADP installation](https://github.com/rh-mobb/documentation/blob/main/content/docs/misc/oadp/rosa-sts/_index.md)
-
-
-<hr style="height:1px;border:none;color:#333;">
-
-<h1 align="center">Debugging Failed Backups<a id="backup"></a></h1>
-
-This section includes steps to debug a failed backup. For more specific issues related to Restic/CSI/volume snapshots check out the following section. Both the OpenShift commands and native Velero commands are useful in troubleshooting.
+1. [OADP FAQ](https://access.redhat.com/articles/5456281)
+1. [OADP Official Troubleshooting Documentation](https://docs.openshift.com/container-platform/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html)
```

Comment:
Should say when to use FAQ vs reading official troubleshooting doc

Reply:
I don't really have a universal rule as long as what we had off has links to official doc and faq.